libtsuba/src/RDGCore.cpp

```diff
@@ -204,15 +204,6 @@ RDGCore::EnsureEdgeTypesLoaded() {
   return katana::ResultSuccess();
 }
 
-void
-tsuba::RDGCore::InitArrowVectors() {
-  // Create an empty array, accessed by Distribution during loading
```

Comment:
So this PR eliminates `InitArrowVectors`?  I thought there was code that assumes these vectors are not null.

Reply:
To be clear: Open was fine with this because it doesn't touch those arrays, but when you would try to load that RDG in enterprise things would blow up.

Reply:
Ok. It sounds like the right thing to do was to allow them to be null, and now all the code does that?  Sorry for initializing them, that looks like it was the wrong path.